src/main/proto/tech/figure/util/v1beta1/types.proto

```diff
@@ -40,8 +43,11 @@ A monetary `amount` in specified `currency`.
 In Java: Use <a href="https://www.joda.org/joda-money/">Joda Money</a>, `BigMoney`, or `BigDecimal` to represent this value.
  */
 message Money {
-  double amount   = 1 [(validate.rules).double.gte = 0]; // Positive amounts only
   string currency = 2 [(validate.rules).string.len = 3]; // ISO 4217 3-digit currency code
+  string value    = 3 [(validate.rules).string.pattern = "^([0-9]+(?:[\\.][0-9]+)?|\\.[0-9]+)$"]; // Positive amounts only, expects digits after a `.`, no +/- signs
```

Comment:
I believe one of the proposed changes was to also allow negative amounts

Reply:
Updated to allow negatives